cmd/iam-store.go

```diff
@@ -1874,8 +1874,8 @@ func (store *IAMStoreSys) ListServiceAccounts(ctx context.Context, accessKey str
 }
 
 var (
-	RegexMappedSysUID = regexp.MustCompile("^uid:([1-9][0-9]*)$")
-	RegexMappedSysGID = regexp.MustCompile("^gid:([1-9][0-9]*)$")
+	RegexMappedSysUID = regexp.MustCompile("^uid:(0|[1-9][0-9]*)$")
```

Comment:
```suggestion
	RegexMappedSysUID = regexp.MustCompile(`^uid:\d+$`)
```

Reply:
I defined `0` separately just to avoid matching of such kind of patterns: `gid:001`. Of course in next steps this should be well parsed by `ParseInt` but I've decided to set 0 as separated match option